pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -159,6 +159,10 @@ func (r *hibernationReconciler) Reconcile(ctx context.Context, request reconcile
 	if !cd.Spec.Installed {
 		return reconcile.Result{}, nil
 	}
+	// If cluster is fake, skip any processing
+	if controllerutils.IsFakeCluster(cd) {
+		return reconcile.Result{}, nil
```

Comment:
What does a consumer end up doing with this reconciliation? If a consumer is creating a claim against a pool, they're likely watching the resulting resource to see it go ready and use the kubeconfig. Will a pool with this fake annotation allow for this flow?